src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/optimizer/utils/CardinalitiesCache.java

```diff
@@ -0,0 +1,23 @@
+package se.liu.ida.hefquin.engine.queryproc.impl.optimizer.utils;
+
+import se.liu.ida.hefquin.engine.queryplan.PhysicalPlan;
+
+import java.util.*;
+
+public class CardinalitiesCache {
+    protected final Map<PhysicalPlan, Integer> map = new HashMap<>();
+
+    public boolean add ( final PhysicalPlan lp, final Integer card ) {
```

Comment:
Why does this method return a boolean?
... and what is the meaning of this return value?
... and why does implementation of this method always return true?